woocommerce/woocommerce-gutenberg-products-block
# Enable several i18n ESLint rules from wpcalypso

Fixes #1678.

In #1672 it was suggested to add an ESLint rule to prevent using three dots instead of ellipsis in translatable strings. This PR adds this ESLint rule and some others from `eslint-plugin-wpcalypso`.

Fortunately there was no need to make any code change because all our code already passes! :tada:  But these rules should prevent us from introducing future bugs.

### How to test the changes in this Pull Request:

1. Run `npm run lint:js`.
2. Verify there are no errors (there is a warning, but unrelated to this PR and it's fixed in #1468).

Docs for each rule can be found in https://github.com/Automattic/wp-calypso/tree/master/packages/eslint-plugin-wpcalypso#supported-rules.
